Removed the space in 'Mark Text' (#2763)

bolshoytoster opened the pull request:
| Q                 | A
| ----------------- | ---
| Bug fix?          | no
| New feature?      | no
| Breaking changes? | As far as I've tested, no
| Deprecations?     | no
| New tests added?  | not needed
| Fixed tickets     | Fixes #2763
| License           | MIT

### Description

[Description of the bug or feature]

Literally just replaced all occurances of 'Mark Text' with 'MarkText' using sed.

Ovilia commented on .github/CHANGELOG.md:
```diff
@@ -188,7 +188,7 @@ v0.15.1 is an unplanned release to fix a XSS security vulnerability.
 - Added portable zip archive for both x86 and x64 Windows.
 - Changed `viewToggleFullScreen` and `windowCloseWindow` key bindings to `windowToggleFullScreen` and `fileCloseWindow`.
 - Removed `viewChangeFont` key binding.
-- Mark Text is now single-instance application on Linux and Windows too.
```
I think changelog should stay unchanged.

bolshoytoster replied:
Fair enough I'll revert that

Ovilia commented on .github/CHANGELOG.md:
```diff
@@ -188,7 +188,7 @@ v0.15.1 is an unplanned release to fix a XSS security vulnerability.
 - Added portable zip archive for both x86 and x64 Windows.
 - Changed `viewToggleFullScreen` and `windowCloseWindow` key bindings to `windowToggleFullScreen` and `fileCloseWindow`.
 - Removed `viewChangeFont` key binding.
-- Mark Text is now single-instance application on Linux and Windows too.
```
I think changelog should stay unchanged. What is released is released.

bolshoytoster commented:
I've reverted the changes in the `.github` directory, including the changelogs.

I've also added an alias `cs` for `csharp`, this should resolve #2760.

bolshoytoster commented:
I've now added an error handler in the render process.

This should resolve #2758

The `src/muya/lib/contentState/clickCtrl.js` file will now return if `document.querySelector` fails to find an element, instead of running into a `TypeError`.

Jocs commented on src/muya/lib/contentState/clickCtrl.js:
```diff
@@ -12,6 +12,9 @@ const clickCtrl = ContentState => {
       const lastBlock = this.getLastBlock()
       const archor = this.findOutMostBlock(lastBlock)
       const archorParagraph = document.querySelector(`#${archor.key}`)
+      if (archorParagraph === null) {
```
Why add these codes?

bolshoytoster replied:
I added this due to a TypeError that can be raised (pointed out in #2758) when `document.querySelector` returns null.

bolshoytoster replied:
I could change it to just `if (!archorParagraph) {`

Jocs commented:
@bolshoytoster Thanks for your contribution.
